### PR TITLE
Expose experiment._sessionCollection

### PR DESCRIPTION
**Pull request by samchrisinger**
So consuming code can check if the session collection gets created properly

### CODE REVIEW

File: exp-models/addon/models/experiment.js [hunk 1]
```diff
@@ -149,6 +153,12 @@ export default DS.Model.extend(JamModel, {
                 [`jam-${config.JAMDB.namespace}:accounts-*`]: 'CREATE'
             }
         });
-        collection.save();
+        this.set('_sessionCollection', collection);
+        collection.save().then(() => {
+            this.set('sessionCollectionId', this.get('sessionCollectionId'));
```
Comment (abought) [inline exp-models/addon/models/experiment.js]: `sessionCollectionId` is a computed, and does not depend on the newly created session. Why are you trying to set it?

File: exp-models/addon/models/experiment.js [hunk 2]
```diff
@@ -122,6 +122,10 @@ export default DS.Model.extend(JamModel, {
         // Eg an experiment called 'test0' would have a collection 'session-test0'
         return `session${this.get('shortId')}s`; // FIXME: the spurious s is a requirement imposed by genschemas...
     }),
+    // Only use to check that the session collection is created when an
+    // experiment document is created. NOT a reliable way to fetch an
+    // experiment's session collection
+    _sessionCollection: null,
```
Comment (chrisseto) [inline exp-models/addon/models/experiment.js]: Any reason to not make this a computed?

Comment (samchrisinger) [inline exp-models/addon/models/experiment.js]: I'm not trying to encourage people to use this as a convenient way to fetch the experiment's sessions collection. This approach, while limited requires much less logic.

File: exp-models/addon/models/experiment.js (hunk 1 above)
Comment (samchrisinger) [inline exp-models/addon/models/experiment.js]: Good point

**Pull request merged**